target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -195,6 +195,19 @@ tplink,archer-mr200)
 tplink,re200-v1)
 	ucidef_set_led_netdev "lan" "lan" "green:lan" "eth0"
 	;;
+wavlink,wl-wn579x3)
+	ucidef_set_led_switch "lan" "lan" "blue:lan" "switch0" "0x2f"
+	ucidef_set_led_switch "wan" "wan" "blue:wan" "switch0" "0x10"
+	# associate the 'wps' led with wan0, the 5GHz interface
+	ucidef_set_led_netdev "wifi0_led" "wifi0" "blue:wps" "wlan0"
+	# associate the 'wifi' led with wan1, the 2GHz interface
+	ucidef_set_led_netdev "wifi1_led" "wifi1" "blue:wifi" "wlan1"
+	# show the rssi for the 5GHz interface
```

Comment:
is this your choice or the vendors?

Reply:
Assuming you are referring to:
>  	# show the rssi for the 5GHz interface 

Yes it was an arbitrary choice between that and wlan1 (2GHz interface). 
IIRC the vendor behaviour is to have user configurable TX strength in the admin section, which is what is shown here.

Do you know if the rssileds configuration supports multiple wlan interfaces? Tying these LEDs to one or both wlan's RSSI feels like a better default experience than doing nothing, but I'm happy to leave out if you prefer?

Reply:
Then please drop the rssileds. They are mostly a pain anyway, since they quite terribly break when interface names change.